### PR TITLE
DHFPROD-5898:Create hub-central roles to be shown in service portal

### PR DESCRIPTION
**Pull request by srinathgit**
### Description
Create hub-central roles to be shown in service portal
#### Checklist: 
```diff
- Note: do not change the below
```

-  ##### Owner:

- [x] JIRA_ID included in all the commit messages
- [x] PR title is in the format JIRA_ID:Title
- [x] Rebase the branch with upstream
- [x] Squashed all commits into a single commit
- [x] Added Tests
  

- ##### Reviewer:

- [x] Reviewed Tests

### CODE REVIEW

File: examples/reference-entity-model/src/main/ml-config/security/users/wailin.json [hunk 1]
```diff
@@ -3,10 +3,6 @@
   "description": "Wai Lin is a Systems Analyst",
   "password": "%%testPassword%%",
   "role": [
-    "hub-central-load-writer",
-    "hub-central-mapping-writer",
-    "hub-central-custom-reader",
-    "hub-central-step-runner",
-    "hub-central-downloader"
+    "hub-central-curator"
```
Comment (rjrudin) [inline examples/reference-entity-model/src/main/ml-config/security/users/wailin.json:6]: Let's keep the downloader role, we need someone who's not a hc-developer that can do this for testing purposes.

File: marklogic-data-hub/src/main/resources/hub-internal-config/security/roles/hub-central-curator.json [hunk 2]
```diff
@@ -0,0 +1,12 @@
+{
```
Comment (rjrudin) [inline marklogic-data-hub/src/main/resources/hub-internal-config/security/roles/hub-central-curator.json:1]: We still need tests for all 3 of these new roles. I think we can get away with just reading the role from the Manage API and verifying that the role inherits all of the expected roles. Each assertion should explain why the role is being inherited.

These should be core tests - can put them in the hub.security package.

**Review by rjrudin — CHANGES_REQUESTED** [review]

File: marklogic-data-hub/src/main/resources/hub-internal-config/security/roles/hub-central-curator.json [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+  "role-name": "hub-central-curator",
+  "description": "Permits managing ingestion, mapping steps, flows and running steps in Hub Central.",
+  "role": [
+    "hub-central-step-runner",
```
Comment (bsrikan) [inline marklogic-data-hub/src/main/resources/hub-internal-config/security/roles/hub-central-curator.json:5]: We can drop this. hc-flow-writer inherits step-runner

**Review by bsrikan — APPROVED** [review]

**Review by ryanjdew — APPROVED** [review]

**Review by rjrudin — APPROVED** [review]

**Comment by srinathgit** [comment]
@SameeraPriyathamTadikonda ,
I think this can be merged. There are 2 cypress test failures, they are not related to the PR.

**Pull request merged**